### ISSUE TITLE
@material/mwc-button/mwc-button.js has wrong import

### ISSUE THREAD

**Issue by andythsu** [material-components/material-components-web-components#169]
error:
`Can't resolve 'lit-html/directives/classMap.js' in 'C:\AndyDev\ui-components\src\app\packages\node_modules\@material\mwc-button`

After going through the mwc-button, it turns out that the import should be 
`import { classMap } from 'lit-html/directives/class-map.js';`

```
"dependencies": {
        "@webcomponents/webcomponentsjs": "latest",
        "@polymer/lit-element": "latest",
        "@material/mwc-button": "latest"
    }
```

**Comment by 43081j** [comment]
Fixed by #167

**Comment by lozandier** [comment]
This still seems to be an issue; some components have `src`, some do not—which is *extremely* confusing for those trying to use the components today.

**Comment by straversi** [comment]
Original issue fixed by fbae6bedd5d8fcf5657e0acb2764a970eec2003c.

As far as components being written in JS; efforts toward converting all components to typescript are ongoing.
